Database-Files/CreateDatabase.sql

```diff
@@ -83,7 +83,6 @@ CREATE TABLE comp_Band (
     compID int NOT NULL,
     bandID int NOT NULL,
     compDesc varchar(200) NOT NULL,
-    PRIMARY KEY(compID, bandID),
```

Comment:
Why are we removing these

Reply:
address these comments!

Reply:
It was preventing us inserting id's into the field and an unnecessary constraint.